ads/uas.md

```diff
@@ -51,7 +51,7 @@ Note that the `width` and `height` mentioned should be maximum of the width-high
 <amp-ad
     width="300" height="250"
     type="uas"
-    json='{"accId": "132109", "adUnit": "10002912", "sizes": [[300, 250]], "targetings": {"country": ["India", "USA"], "car": "Civic"}, "locLat": "12.24", "locLon": "24.13", "locSrc": "wifi", "pageURL": "http://mydomain.com"}'>
+    json='{"accId": "132109", "adUnit": "10002912", "sizes": [[300, 250]], "targetings": {"country": ["India", "USA"], "car": "Civic"}, "locLat": "12.24", "locLon": "24.13", "locSrc": "1", "pageURL": "http://mydomain.com"}'>
```

Comment:
I didn't find the supported value for `locSrc`. Is `1` supported? `wifi` might actually make sense for "location source"

Reply:
Hello @lannka ,

Thank you for your suggestion.
PubMatic UAS tag needs to pass the value for locSrc parameter as per a pre-defined map of values thus we are changing it from "wifi" to "1".

Reply:
OK, that sounds good. But if you folks a the maintainer of PubMatic, I suggest you provide the full map in the documentation as well.